homeassistant/data_entry_flow.py

```diff
@@ -435,6 +431,16 @@ async def _async_handle_step(
 
         return result
 
+    def _raise_if_not_has_step(self, flow: FlowHandler, step_id: str) -> None:
```

Comment:
_raise_if_step_missing or _raise_if_step_does_not_exist. Current name is weird :)